libs/client-shared/src/lib/components/select/select.component.ts

```diff
@@ -31,21 +34,26 @@ import { SmartTranslatePipe } from '../smart-translate.pipe';
     MatFormField,
     SmartTranslatePipe,
     MatHint,
+    FormItemWrapperComponent,
   ],
 })
 export class SelectComponent<T> implements OnInit, ControlValueAccessor {
   @Input() public values: T[] = [];
   @Input() public bindLabel: keyof T | null = null;
+  @Input() public bindKey: keyof T | null = null;
   @Input() public title = '';
+  @Input() public icon = '';
   @Input({ transform: coerceBooleanProperty }) public multiple = false;
   @Input() public initialValues: T[] = [];
   @Input() public shouldShowError = false;
   @Input() public errorMessage = '';
+  @Input({ transform: coerceBooleanProperty }) public shouldShowTrigger = false;
+  @Input() triggerLabel = '';
```

Comment:
Why do we need the `shouldShowTrigger` here? Can't we just check if `trigger` is set? (same with `error`, btw).
```suggestion
  @Input() trigger = '';
```
If you want to make it possible to show the trigger counter without a specific label, you could do something like this:
```ts
@Input({ transform: (value: unknown) => value === '' ? true : value })
@Input trigger: string | boolean | null = null
```
This should allow you to display the trigger when `!!trigger` and the label only if `trigger !== true`.
On the use side it would look something like this:
```html
<asset-sg-select ... trigger />
```